base/distributed/managers.jl

```diff
@@ -470,7 +470,7 @@ function socket_reuse_port()
             # provide a clean new socket
             return TCPSocket()
         end
-        is_apple() && bind_client_port(s)
+        is_bsd() && bind_client_port(s)
```

Comment:
Just to ensure that if in the future,  Windows starts supporting reuseport, and requires early binding, we then have to change only the first line.